cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -159,8 +159,7 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 				continue
 			}
 
-			/* TODO: communicate back the AMI */
```

Comment:
But, the TODO isn't done ;)

Reply:
Is it still needed though? Like now the thing shares?

Reply:
Yeah, we still need to tell the caller which AMI was shared with them.